dashboard-dataviz/zimbabwe_dashboard/app.R

```diff
@@ -1,28 +1,1948 @@
-# Zimbabwe Movement Dashboard
+# Zimbabwe Mobility Dashboard
 
-# PACKAGES AND SETUP ===========================================================
+##### ******************************************************************** #####
+# 1. PACKAGES AND SETUP ========================================================
 
+#### R Shiny Deployment Options
 options(rsconnect.max.bundle.files = 300000)
-# options(rsconnect.max.bundle.size = 90000000)
-
-
 
 #### Setting directory so will work locally
 if (Sys.info()[["user"]] == "robmarty") {
   setwd("~/Documents/Github/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard")
 }
 
 if (Sys.info()[["user"]] == "WB521633") {
-  setwd(
-    "C:/Users/wb521633/Documents/Github/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
+  setwd("C:/Users/wb521633/Documents/Github/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
   )
 }
 
 if (Sys.info()[["user"]] == "wb519128") {
-  setwd(
-    "C:/Users/wb519128/GitHub/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
+  setwd("C:/Users/wb519128/GitHub/covid-mobile-data/dashboard-dataviz/zimbabwe_dashboard"
   )
 }
 
```

Comment:
Did you move the dashboard back to just one file or did you branch before it was splited?

Reply:
I'm fairly sure I branched before I combined everything back into one

Reply:
And yes, this reflects the published version